scripts/bestarch/id.json/id.json

```diff
@@ -0,0 +1 @@
+[161,247,66,57,203,188,141,236,124,123,200,192,255,23,161,34,116,202,70,182,176,94,195,68,185,32,61,42,203,57,245,190,153,233,189,58,187,209,15,232,83,112,145,116,89,187,201,44,36,255,81,102,84,101,62,1,180,217,194,37,147,74,7,170]
```

Comment:
lets not copy all the keys
we have it in evm repo

Reply:
done